src/components/initiator.py

```diff
@@ -56,7 +58,7 @@ def __init__(self, transport, initiator, peer, trace, initiatorAddress, peerAddr
         self.intervalMin        = 24; # Minimum Connection interval := 24 x 1.25 ms = 30 ms
         self.intervalMax        = 24; # Maximum Connection interval := 24 x 1.25 ms = 30 ms
         self.latency            = 0;  # Don't allow peer to skip any connection events
-        self.supervisionTimeout = 25; # Supervision timeout := 25 x 10 ms = 250 ms.
+        self.supervisionTimeout = self.supervisionTimeoutDefault
```

Comment:
If dropping the `max` part in the calculations below, this could just use a parameter with default value.